packages/core/src/hub.ts

```diff
@@ -361,7 +361,17 @@ export class Hub implements HubInterface {
    * @inheritDoc
    */
   public startTransaction(context: TransactionContext, customSamplingContext?: CustomSamplingContext): Transaction {
-    return this._callExtensionMethod('startTransaction', context, customSamplingContext);
+    const result = this._callExtensionMethod<Transaction>('startTransaction', context, customSamplingContext);
+
+    if (__DEBUG_BUILD__ && !result) {
+      throw new Error(`Tracing extension 'startTransaction' has not been added. Call 'addTracingExtensions' before calling 'init':
```

Comment:
l: Can we somehow combine these two into a util to generate the string? So we can avoid having these almost-same strings twice in the build output. E.g.:

```js
function _getTracingExtensionsErrorMessage(methodName) {
  return `Tracing extension '${methodName}' has not been added.
Call 'addTracingExtensions' before calling 'init':
Sentry.addTracingExtensions();
Sentry.init({
  ...
});`;
}
```

Reply:
Ah, this is a mistake, it should only appear once!